packages/mongodb-runner/src/mongocluster.spec.ts

```diff
@@ -0,0 +1,172 @@
+import { expect } from 'chai';
+import { MongoCluster } from './mongocluster';
+import { promises as fs } from 'fs';
+import path from 'path';
+import os from 'os';
+import createDebug from 'debug';
+
+if (process.env.CI) createDebug.enable('mongodb-runner');
+
+const twoIfNotWindowsCI =
+  process.platform === 'win32' && process.env.CI ? 0 : 2;
+
+describe('MongoCluster', function () {
+  this.timeout(120_000);
+
+  let tmpDir: string;
+  let cluster: MongoCluster;
+
+  before(async function () {
+    tmpDir = path.join(os.tmpdir(), `runner-tests-${Date.now()}`);
+    await fs.mkdir(tmpDir, { recursive: true });
+  });
+
+  after(async function () {
+    await fs.rm(tmpDir, {
+      recursive: true,
+      maxRetries: 100,
+    });
+  });
+
+  afterEach(async function () {
+    await cluster?.close();
```

Comment:
Nitpick, but if a test fails at cluster = await MongoCluster.start()  then won't afterEach will re-close the previous cluster? Kinda inconsequential because a test would have already failed.

Reply:
Actually nevermind. I see close() clears out all the servers anyway.